theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
`@sk-premium-job-highlight` is too specific to SERP I think, what if we want to use the same colour scheme in company reviews for example. Should the color name define its use?